lib/internal/test_runner/test.js

```diff
@@ -681,6 +681,16 @@ class Test extends AsyncResource {
     }
   }
 
+  wasAbortedByUser() {
+    return this.#outerSignal?.aborted;
```

Comment:
I think we should make `#outerSignal` and `#abortController` not private to make this cleaner

Reply:
done